tests/quickstart_test.py

```diff
@@ -0,0 +1,87 @@
+import os
+import time
+import pytest
+from testbook import testbook
+import json
+from pytest_dependency import depends  # import the depends function
+from common_test import gcp_setup
+
+
+@pytest.fixture(scope="module")
+def tb():
+    with testbook("docs/quickstart.ipynb", execute=False, timeout=1800) as tb:
+        yield tb
+
+
+@pytest.fixture(scope="module")
+def auth_tb(tb):
+    cred_file_path = f"{os.path.expanduser('~')}/.gcp/doc-test-creds.json"
+    assert os.path.isfile(cred_file_path)
+    with open(cred_file_path, "r") as f:
+        creds_json = json.loads(f.read())
+        project_id = creds_json["project_id"]
+    tb.inject(
+        code=f"!gcloud auth activate-service-account --key-file={cred_file_path} -q"
```

Comment:
once we move this to Github actions we can use default credentials and Workload Identity Pools: https://cloud.google.com/blog/products/identity-security/enabling-keyless-authentication-from-github-actions

Reply:
definitely! creating an issue for it now